gobblin-runtime/src/main/java/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -107,6 +111,8 @@ public MetricContext apply(JobMetrics input) {
             return input.getMetricContext();
           }
         });
+
+    this.eventSubmitter = new EventSubmitter.Builder(this.runtimeMetricContext, "gobblin.runtime").build();
```

Comment:
As per our offline discussion this is how I think API would be easier for the consumers.

```java
public class MetricContext {

// The reporting class will be the default tag/namespace for metrics and events
public MetricContext (Class reportingClass)

// I can also set namespace using additional tags
public MetricContext (Class reportingClass, Map<String, String> additionalTags)

// Fire single event
public void sendEvent(String eventName)

// Fires single event with key/values in the map, calls toString and serializes the values
public void sendEvent(Map<String, Object> events)

}
```